### PR TITLE
Update README to use new remote plugins

### PR DESCRIPTION
**Pull request by pkwarren**
#### Brief description of what is fixed or changed

Update the example buf.gen.yaml in the README to use the new remote plugins.

### CODE REVIEW

**Review by johanbrandhorst — CHANGES_REQUESTED** [review]
This is great, do you mind updating our own `buf.gen.yaml` too? https://github.com/grpc-ecosystem/grpc-gateway/blob/main/buf.gen.yaml

File: Makefile [hunk 1]
```diff
@@ -82,7 +82,7 @@ $(GENERATE_UNBOUND_METHODS_EXAMPLE_SRCS): $(GENERATE_UNBOUND_METHODS_EXAMPLE_SPE
 		$(EXAMPLE_CLIENT_DIR)/generateunboundmethods/git_push.sh
 
 install:
-	go install github.com/bufbuild/buf/cmd/buf@v1.3.1
+	go install github.com/bufbuild/buf/cmd/buf@v1.12.0
```
Comment (pkwarren) [inline Makefile:85]: New `plugin:` key in buf.gen.yaml requires Buf CLI 1.8+: https://docs.buf.build/bsr/remote-plugins/migrating-from-alpha#bufgenyaml-changes.

**Review by johanbrandhorst — APPROVED** [review]

**Comment by johanbrandhorst** [comment]
Looks like the buf update changed the formatting, do you mind taking another look?

**Comment by johanbrandhorst** [comment]
I think this was superceded

**Pull request closed without merging**